tests/speech-to-text/sanity_stt_git_test.py

```diff
@@ -21,28 +21,30 @@
 logger = logging.getLogger(__name__)
 
 unique_id = get_unique_id()
+test_storage_git = StorageGit(
+    repo=GitRepo(
+        name=f"STT-RoboShaulTiny-dataset{unique_id}",
+        repository_url="https://huggingface.co/datasets/hirundo-io/RoboShaulTiny",
+        plain_auth=GitPlainAuthBase(
+            username="blewis-hir",
+            password=os.environ["HUGGINGFACE_ACCESS_TOKEN"],
+        ),
+    ),
+    branch="main",
+)
 test_dataset = OptimizationDataset(
     name=f"TEST-STT-RoboShaulTiny-dataset{unique_id}",
-    labelling_type=LabellingType.SPEECH_TO_TEXT,
+    labeling_type=LabelingType.SPEECH_TO_TEXT,
```

Comment:
did you fix here to American writing 'labeling'? we are British and so are you

Reply:
Lol :P